packages/plugins/i18n/admin/src/components/CMEditViewInjectedComponents/CMEditViewLocalePicker/index.js

```diff
@@ -72,6 +72,11 @@ const CMEditViewLocalePicker = ({
     });
   };
 
+  const handleChangeAndReloadPage = (value) => {
+    handleChange(value);
+    location.reload();
```

Comment:
Why are we reloading? Can you write a comment please.

Reply:
I don't think we should need to reload the page at all for this, the system should be able to handle this sort of thing. 
You're using `window.location.href` and accessing `searchParams` you should be using `react-router-dom`'s hooks that will update with URL state changes imo.

Reply:
We also have `usePluginsQueryParams`